llvm/lib/Transforms/Instrumentation/HWAddressSanitizer.cpp

```diff
@@ -1542,8 +1557,13 @@ void HWAddressSanitizer::sanitizeFunction(Function &F,
     }
   }
 
+  DominatorTree &DT = FAM.getResult<DominatorTreeAnalysis>(F);
+  PostDominatorTree &PDT = FAM.getResult<PostDominatorTreeAnalysis>(F);
+  DomTreeUpdater DTU(DT, PDT, DomTreeUpdater::UpdateStrategy::Lazy);
+  LoopInfo &LI = FAM.getResult<LoopAnalysis>(F);
```

Comment:
We could pull the definitions for DT PDT and LI out of the if in 1538 then we need not repeat them.

Reply:
Done, thanks!